src/core/public/kbn_bootstrap.ts

```diff
@@ -60,6 +66,22 @@ export function __kbnBootstrap__() {
         setup.fatalErrors.add(i18nError);
       }
 
-      await coreSystem.start();
+      const start = await coreSystem.start();
+
+      if (APM_ENABLED && start) {
```

Comment:
NIT: unnecessary `&& start`, as the return from `coreSystem.start()` is not optional.

Reply:
`coreSystem.start` would return undefined if there is error thrown, so added this guard.